controllers/collections.js

```diff
@@ -124,7 +124,7 @@ exports.replaceById = async function (req) {
   data._id = req.params.id
   data.meta = data.meta || {}
   data.meta.created = (oldDoc.meta || {}).created
-  data.meta.owner = (oldDoc.meta || {}).owner
+  data.meta.owner = data.meta.owner || (oldDoc.meta || {}).owner
```

Comment:
After thinking a bit, I think a new permission isn't necessary since if you can edit a doc (either an owner or admin) then I feel you can probably transfer ownership. Either way though, this code needs some updates. The  `notify('put'` below eventually calls the [collectionPermissionCheck](https://github.com/thomas4019/expressa/blob/93cdad4a35f72d397082bc22fd49baab2691ca51/listeners_collection_permissions.js) with the *new* document and looks at the owner on it. So I think it makes sense to keep the owner override here so the permission check works.

But we can store the new owner, if set, in a temporary variable and then set in on the doc on the life after the `notify('put'`

Reply:
i can think of a few scenarios where someone could malicously change owner of their own doc to a say a a rival. So will put not rush this for now given current implementation is safest.

Lets say we added a permission `{collection} edit owner`. For consistentcy the listeners should receive the updated owner, so changing after the notify wont work. Would be best to add a new listener afer `collectionPermissionCheck` in file `listeners_collection_permissions.js` which would enforce this permission. What other updates would be required?